.github/workflows/pr_open.yml

```diff
@@ -108,7 +133,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: latest
-          args: -v --timeout 10m -E gofmt,gci
+          args: --fix
```

Comment:
I think auto fix in CI is dangerous.
Why remove gofmt and cgi?

Reply:
I agree!

Reply:
The IC doesn't push the changes on the branches, there won't be any problem :) 

I used the configuration file `.golangci.yml` instead of the command line arguments